projects/perennial-v2/index.js

```diff
@@ -6,6 +6,11 @@ const config = {
     factory: '0xDaD8A103473dfd47F90168A0E46766ed48e26EC7',
     fromBlock: 135921706,
   },
+  perennial: {
+    dsu: '0xC27399bE9E39f7F6b1f94fBd512F5c2aD2b5eDb7',
```

Comment:
this doesnt seem like an erc20, is that right?

Reply:
Apologies, you are correct. This is fixed in this commit: https://github.com/equilibria-xyz/DefiLlama-Adapters/commit/0dd46649f5bb83880dae30a8f2c23ca1cc42e5b2

But I think since this PR is closed it's not syncing?

Reply:
0x7b4Adf64B0d60fF97D672E473420203D52562A84 also doesnt seem like an erc20

Reply:
we need DSU to be an erc20 for us to price it

Reply:
Here's the contract code https://explorer.perennial.foundation/token/0x7b4Adf64B0d60fF97D672E473420203D52562A84?tab=contract

It uses OZ's standard ERC20 - is there something that's missing there?

Reply:
my mistake, not sure what I was looking at, this should be priced now and show in TVL stats in the next hour or so